runtime/lib/component/custodian.js

```diff
@@ -168,6 +175,12 @@ Custodian.prototype.prepareNetwork = function prepareNetwork () {
   }
   if (wifi.getNumOfHistory() > 0) {
     logger.info('has histroy wifi, just skip')
+    if (this.promptTimer !== null) {
+      clearTimeout(this.promptTimer)
+    }
+    this.promptTimer = setTimeout(player => {
+      player.ttsSound('@yoda', 'system://guide_config_network.ogg')
```

Comment:
light effect `system://boot.js` should also be stopped here in order to resolve https://bug.rokid-inc.com/zentaopms/www/index.php?m=bug&f=view&bugID=18835

Reply:
The `system://boot.js` light will be stopped by `guide_config_network.ogg` when play `light.ttsSound` due to there is followed speech light effect.
Should it still need `stop` it manually?